data_utils/helper_functions.py

```diff
@@ -33,9 +37,15 @@ def order_candidates(candidate_neighbours: dict, distance_matrix: [[float]]):
         Matrix containing the distances for each pair of atoms within the structure.
     """
 
-    print("Ordering candidate neighbours based on their distance.")
+    print_distributed(
+        verbosity, "Ordering candidate neighbours based on their distance."
+    )
     sorted_candidate_neighbours = {}
-    for point_index, candidates in tqdm(candidate_neighbours.items()):
+    for point_index, candidates in (
+        tqdm(candidate_neighbours.items())
+        if ((0 == dist.get_rank() and 2 == verbosity) or 4 == verbosity)
```

Comment:
Can you make this another `print_utils` function? Nice to keep it all in one place

Reply:
@streeve Done